File: Server/Server/Models/Requests/UserValidationRequest.cs
namespace Server.Models.Requests
{
    public class UserValidationRequest
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }

        public UserValidationRequest()
        {
            Id = 0;
            Email = string.Empty;
            Name = string.Empty;
        }
    }
}

File: Client/app/src/main/java/com/example/client/networking/requests/UserValidationRequest.java
package com.example.client.networking.requests;

public class UserValidationRequest {
    private String email;
    private int id;
    private String name;

    public UserValidationRequest(String email, int id, String name) {
        this.email = email;
        this.id = id;
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
